contrib/prometheus/README.md

```diff
@@ -28,6 +27,13 @@ prometheus-retention-time = 15
 # Check out the documentation for the list of available metrics.
 prometheus = true
 ```
+
+Test manually at:
+`http://localhost:1317/metrics?format=prometheus`
```

Comment:
Wait, we need the lcd server to display metrics?
Sounds like a left-over design. We should revisit this at some point, but fine for this PR.

Reply:
ja, there is another endpoint at port `26660` in `config.toml` but it took me a bit to figure out why `1317` was not returning any of our data (because of the missing format param)